docs/team/kvignesh122.md

```diff
@@ -0,0 +1,62 @@
+# Vignesh's Project Portfolio Page
+
+## Project Overview: mTracker
+
+mTracker is a **CLI-based trading journal** that allows
+user to store and view important trading-related information on their
+shortlisted financial instruments. The program allows users to store
+information on 4 types of instruments: Stock, ETF, Forex, Cryptocurrency. 
+mTracker is written in Java 11 and contains over 5000 LoC.
+
+### My key contributions to the project:
+
+- Tested out various real-time market data API provider for initial idea of 
+  implementing real-time capabailities in program. Coded and tried out a HTTP client
+  server in Java. Idea was later discarded by team for various reasons.
+- Project was mainly divided among group members based on instrument types.
+I was in charge of implementing all forex-related command and parser classes, and their testing.
+- Implemented the find feature, which allows users to search for 
+instruments saved in the watchlist using their input search term.
+- Was instrumental in beautifying the program's UI. Coded the ACSII patterns
+for bye, and mTracker logo at startup.
+- Made sure that error messages were consistent in formatting with fullstops 
+and commas. Also made sure grammar is correct for the error messages and the UG due to 
+  my strength in language.
+- Coded the main run method in MTracker class and also the bye command for exiting
+program.
+- Abstracted out IndexedCommand class to handle index input by user, and
+managing it alongside view, delete, done, and edit command classes.
+- Abstracted out AddInstrumentParserTest class for all other instruments' addition parser test classes
+to depend on.
+- Improved overall code quality by abstracting out repetitive codeblocks as
+  methods, and removed redundant setters and getters.
```

Comment:
Can shift this into team based task
https://nus-cs2113-ay2122s1.github.io/website/schedule/week12/project.html#ppp-sections-to-include-2

Reply:
Ok